libs/pages/elewa/home/src/lib/features-elewa-home.module.ts

```diff
@@ -15,25 +15,34 @@ import { PrevDirective } from './directives/prev.directive';
 import { NextDirective } from './directives/next.directive';
 
 import { HomeRoutingModule } from './home.routing';
-
-
+import { ImpactComponent } from './components/impact/impact.component';
+import { ButtonsModule } from '@elewa-group/features/components/buttons';
```

Comment:
Order of imports

Reply:
resolved by grouping components together and modules together